packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -120,7 +120,6 @@ export class Flow extends LitElement implements LoadableComponent {
 
   override connectedCallback(): void {
     this.itemMutationObserver?.observe(this.el, { childList: true, subtree: true });
-    this.handleMutationObserverChange();
```

Comment:
I think this should be fine since connecting a flow doesn't need to update items. The mutation observer should suffice when items are added. If we find an issue we can call on connectedCallback and loaded.